Form/Extension/DateTypeExtension.php

```diff
@@ -59,7 +59,7 @@ public function finishView(FormView $view, FormInterface $form, array $options)
     /**
      * {@inheritdoc}
      *
-     * @todo Remove it when bumping requirements to SF 2.7+
+     * @deprecated Remove it when bumping requirements to SF 2.7+
```

Comment:
@phiamo we could also start a 2.7+ branch and tag the current one as 3.0 as we intended. Merge PRs into the 3.0 branch and then merge into the 2.7+ branch. We're going to have to eventually and 2.7 is supposed to be compatible with 3.0. We can just have travis check up to 2.6 with the old branch and 2.7+ with the new. 2.7 is new LTS as well so it's a good jump imo. The old branch will still work but we won't support the tests for 2.7+, anybody who uses it will still get all the configureOptions etc warnings that come along.